src/lobby/index.ts

```diff
@@ -0,0 +1,64 @@
+import * as core from '../core'
+import * as dns from '../dns'
+import { dataRoot } from '../data-root'
+
+import { USERNAME_BLOCKLIST } from './blocklist'
+
+
+/**
+ * Create a user account.
+ */
+export const createAccount = async (
+  userProps: { email: string, username: string },
+  apiEndpoint?: string,
```

Comment:
maybe wrap this in a `props` object similar to `updateDataRoot`. so a user could supply `apiDid` and not `apiEndpoint` (altho I'm not sure if that'd ever happen :thinking:)

Reply:
Oh good catch, will do.